common4j/src/main/com/microsoft/identity/common/java/nativeauth/providers/NativeAuthOAuth2Configuration.kt

```diff
@@ -51,7 +50,7 @@ class NativeAuthOAuth2Configuration(
 
     companion object {
         //Base url for the mock API to make Native Auth calls. See the swagger at
-        // https://native-auth-mock-api.azurewebsites.net/doc#/ for all possible urls
+        // $(MOCK_API_URL)/doc#/ for all possible urls
         private const val MOCK_API_URL_WITH_NATIVE_AUTH_TENANT = "https://native-auth-mock-api.azurewebsites.net/lumonconvergedps.onmicrosoft.com"
```

Comment:
Mock API value is still set here. Can you please do a ctrl-F in the entire project for this API value string and remove all occurrences?

Reply:
I want to first let the PR passed first using the concrete string since it's not a test file.